doc/maintaining/configuration.rst

```diff
@@ -1556,6 +1556,21 @@ Default value: (none)
 
 By default, the locales are searched for in the ``ckan/i18n`` directory. Use this option if you want to use another folder.
 
+.. _ckan.timezone:
+
+ckan.timezone
```

Comment:
another naming comment :smile: how about "ckan.display_timezone" or similar to make clear how this value is used.